code/controllers/subsystem/starmap.dm

```diff
@@ -20,7 +20,7 @@ SUBSYSTEM_DEF(starmap)
 	var/datum/planet/to_planet
 	var/in_transit_planet // In transit between planets?
 
-	var/is_loading = 0
+	var/is_loading = 0 // 0 = not loading. 1 = loading/unloading new z levels. 2 = recently ran, waiting for she ship to leave FTL
```

Comment:
Use defines for this
for example

#define FTL_NOT_LOADING 1
#define FTL_LOADING 2
#define FTL_RECENT_FTL 3

then undefine them when you stop using them (bottom of the file most likely) by doing

#undefine FTL_NOT_LOADING 1
#undefine FTL_LOADING 2
#undefine FTL_RECENT_FTL 3